### DISCUSSION TITLE
libbpf - 1.4.0 编译报错，无法通过

### DISCUSSION THREAD

**Discussion by chenxuan5858** [immortalwrt/immortalwrt#1265] [Q&A]
### Describe the bug

libbpf - 1.4.0 编译报错，无法通过

### ImmortalWrt version

main

### ImmortalWrt release

main

### ImmortalWrt target/subtarget

main

### Device

X86

### Image kind

Official downloaded image

### Steps to reproduce

![bd34164e255dfa58f9cbdc9b0c44314](https://github.com/immortalwrt/immortalwrt/assets/33675890/2184949e-19a5-42b9-bf84-47a656324288)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by chenxuan5858** [comment]
make dirclean 重新编译解决

**Comment by initzhu** [comment]
扬了tmp 再来，问题解决
